src/index.js

```diff
@@ -636,7 +641,8 @@ class ReactNativeModal extends Component {
             { zIndex: 2, opacity: 1, backgroundColor: 'transparent' },
           ]}
         >
-          {hasBackdrop && backdrop}
+          {!hasCustomBackdrop && hasBackdrop && touchableBackdrop}
```

Comment:
There's a minor typo here, `touchableBackDrop` was created with a capital D, but is referenced as `touchableBackdrop`